# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xmms2/xmms2-0.7DrNo-perl-vendor.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Install perl modules into vendor directory

--- xmms2-0.7DrNo/wafadmin/Tools/perl.py.orig	2011-05-22 01:02:12.236002363 +0200
+++ xmms2-0.7DrNo/wafadmin/Tools/perl.py	2011-05-22 01:02:29.992002362 +0200
@@ -103,7 +103,7 @@
 	conf.env["EXTUTILS_TYPEMAP"] = read_out(" -MConfig -e'print \"$Config{privlib}/ExtUtils/typemap\"'")
 
 	if not getattr(Options.options, 'perlarchdir', None):
-		conf.env["ARCHDIR_PERL"] = Utils.cmd_output(perl + " -MConfig -e'print $Config{sitearch}'")
+		conf.env["ARCHDIR_PERL"] = Utils.cmd_output(perl + " -MConfig -e'print $Config{vendorarch}'")
 	else:
 		conf.env["ARCHDIR_PERL"] = getattr(Options.options, 'perlarchdir')
 
